articles/api-auth/blacklists-vs-grants.md

```diff
@@ -0,0 +1,51 @@
+---
+description: This document covers blacklists vs. grants when it comes handling tokens.
+---
+
+# Blacklists and Client Grants
+
+Let's say that you're using a non-interactive [client](/clients) to represent your protected resource. You have a partner that calls your API, and at the end of your existing contract, you and your partner decide not to renew your partnership. As such, you now want to remove your partner's access to your API. The issue, however, is that you've given your partner an [access token](/tokens/access-token) that lasts for a month.
```

Comment:
@khorne3 I'm a bit confused by the first sentence `Let's say that you're using a non-interactive [client](/clients) to represent your protected resource`: isn't the API the protected resource?